definitions/v3/blutopia-api.yml

```diff
@@ -64,10 +64,26 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    $raw: "{{ range .Categories }}categories[]={{.}}&{{end}}{{ if .Config.freeleech }}freeleech=1&{{ else }}{{ end }}"
+    sortField: "{{ .Config.sort }}"
+    sortDirection: "{{ .Config.type }}"
+    perPage: 100
+    page: 1
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ .Keywords }}"
+    tmdbId: "{{ .Query.TMDBID }}"
+    tvdbId: "{{ .Query.TVDBID }}"
+    imdbId: "{{ .Query.IMDBIDShort }}"
+    # Supported with UNIT3d v6 only
+    seasonNumber: "{{ .Query.Season }}"
+    # Supported with UNIT3d v6 only
+    episodeNumber: "{{ .Query.Ep }}"
```

Comment:
```suggestion
    api_token: "{{ .Config.apikey }}"
    name: "{{ .Keywords }}"
    $raw: "{{ if .Query.Season }}&seasonNumber={{ .Query.Season }}{{ else }}{{ end }}{{ if .Query.Ep }}&episodeNumber={{ .Query.Ep }}{{ else }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
    sortField: "{{ .Config.sort }}"
    sortDirection: "{{ .Config.type }}"
    perPage: 100
    page: 1
```

Well crap! I'd been doing testing with all unused variables removed, so as to 'not complicate things', or so I told myself. Turns out the only variable allowed to have no value is `name`. When we were using incorrectly named variables it didn't cause an issue.

Moved everything necessary to $raw. Restructured to make URL more easily readable in debug/trace logs (but also because having all variables in $raw start with `&` makes my brain happy).

This is _actually_ the last change, in large part because I've lost the will to live ;P

Reply:
I thought the whole purpose was to get away from the lengthy and hard to read arguments and switch to using inputs?